nni/nas/benchmarks/nlp/query.py

```diff
@@ -0,0 +1,63 @@
+import functools
+
+from peewee import fn
+from playhouse.shortcuts import model_to_dict
+from .model import NlpTrialStats, NlpTrialConfig
+
+def query_nlp_trial_stats(arch, dataset, reduction=None, include_intermediates=False):
+    """
+    Query trial stats of NLP given conditions.
+
+    Parameters
+    ----------
+    arch : dict or None
+        If a dict, it is in the format that is described in
+        :class:`nni.nas.benchmark.nlp.NlpTrialConfig`. Only trial stats matched will be returned.
+        If none, all architectures in the database will be matched.
+    dataset : str or None
+        If specified, can be one of the dataset available in :class:`nni.nas.benchmark.nasbench201.Nb201TrialConfig`.
+        Otherwise a wildcard.
+    reduction : str or None
+        If 'none' or None, all trial stats will be returned directly.
+        If 'mean', fields in trial stats will be averaged given the same trial config.
+        Please note that some trial configs have multiple runs which make "reduction" meaningful, while some may not.
+    include_intermediates : boolean
+        If true, intermediate results will be returned.
```

Comment:
also tell users what is its default value

Reply:
Just confused, why do we need to add "what is its default value"(for reduction, it is None but not mean), since it is obviously written in the API (def query_nlp_trial_stats(arch, dataset, reduction=None, include_intermediates=False) ?

Reply:
yes, you are right, no need to write default value